sharing/present.go

```diff
@@ -124,11 +124,13 @@ func (handler *Handler) showObject(ctx context.Context, w http.ResponseWriter, r
 	// on, otherwise we fall back to what wrapDefault was.
 	wrap := queryFlagLookup(q, "wrap",
 		!queryFlagLookup(q, "view", !pr.wrapDefault))
+	// flag for media previews.
+	previewFlag := queryFlagLookup(q, "preview", false)
```

Comment:
i don't understand why this needs to happen?

Reply:
yes, we don't need it. `?wrap=0`  do the same thing. Thank you for the review